site/src/sphinx/zookeeper.rst

```diff
@@ -0,0 +1,58 @@
+.. _`Apache ZooKeeper`: https://zookeeper.apache.org/
+.. _`com.linecorp.armeria.client.zookeeper`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/package-summary.html
+.. _`com.linecorp.armeria.server.zookeeper`: apidocs/index.html?com/linecorp/armeria/server/zookeeper/package-summary.html
+.. _`Mode`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/ZooKeeperEndpointGroup.Mode.html
+
+Automatic service discovery with ZooKeeper
+===========================================
+You can put your Endpoints information into a zNode in `Apache ZooKeeper`_ cluster, as a node value or as the child values of a node, like the following:
+
+| as node value:
+|         /myProductionEndpoints --> 192.168.1.10:8080:100,192.168.1.11:8080:100
```

Comment:
How about removing `:100` for `192.168.1.10` to show what a node value looks like without weight?